server/server.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+
+const textSnippet = require('../api/textSnippet.js');
+app.use(cors({
+    origin: 'http://localhost:3000',
+}));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+app.use('/api', textSnippet);
+
+app.get('/', (req, res) => {
+    res.send('Hello World');
+})
```

Comment:
Need to talk about about how to serve react app through this server instead of a webpack that you user right now.